chapters/background.tex

```diff
@@ -1,4 +1,11 @@
 \section{Background} \label{sec:background}
 % identifying existing approaches within the research literature; this may potentially include mobile-based and non-mobile approaches if necessary
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
-
+TensorFlow is a open source Machine Learning library developed by Google.
+It supports a wide variety of programming languages, mainly targeting web and mobile applications.
+Regarding soHappy we were looking for a lightweight open source approach to detect smiles. Since HaarCascade uses OpenCV which is great for detecting a user's face but struggles when it comes to smile/emotion detection, our choice fell to TensorFlow.
```

Comment:
- Based on my state of knowledge, it's not proper to use 'we' etc. in a paper. And I actually think it's not formal enough. Although in the Remember to smile paper 'we' is used (very few).
So probably it's up to us, and in that cause I vote for not using this personal form in the report. We can also ask Leo about that...
- 'great' is very informal and imprecise (do you mean effective, fast, robust?)
- same with struggles... I suggest to find a more precise description
- in my understanding tensor flow covers another functionality part of the app, so I would make that more clear instead of comparing it like that with haar cascade